libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -583,6 +583,20 @@ bool IRGeneratorForStatements::visit(BinaryOperation const& _binOp)
 	return false;
 }
 
+bool IRGeneratorForStatements::visit(FunctionCall const& _functionCall)
+{
+	FunctionTypePointer functionType = dynamic_cast<FunctionType const*>(&type(_functionCall.expression()));
+	if (
+		functionType &&
+		functionType->kind() == FunctionType::Kind::Internal &&
+		!functionType->bound() &&
```

Comment:
Does this mean that all bound functions go through the dispatch routine?

Reply:
No. They just don't have to be added to the list of direct calls because `endVisit(MemberAccess)` short-circuits for bound function calls and does not add them there.